src/goto-programs/validate_goto_model.h

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************\
+Module: Validate Goto Programs
+
+Author: Diffblue
+
+Date: Oct 2018
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_VALIDATE_GOTO_MODEL_H
+#define CPROVER_GOTO_PROGRAMS_VALIDATE_GOTO_MODEL_H
+
+#include <util/validate.h>
+
+class goto_model_validation_optionst final
+{
+public:
+  bool entry_point_exists;
+  bool function_pointer_calls_removed;
+  bool check_returns_removed;
+  bool check_called_functions;
+
+  // this option is always set to false until all source locations
+  // are reliably set correctly (future work)
```

Comment:
I think the comment below is sufficient, no need to single it out here as well.

Reply:
Done.